app/models/location.rb

```diff
@@ -11,6 +11,8 @@ class Location < ApplicationRecord
   accepts_nested_attributes_for :addresses, reject_if: lambda { |addr| addr[:address].blank? }, allow_destroy: true
 
   validates :name, presence: true
```

Comment:
Yeah i know, I wasn't done completely with location tests, but just wanted to check there were no merge conflicts with what you guys had done this week so i just decided to push